JavaAssignments/src/amarja/Addition.java

```diff
@@ -0,0 +1,48 @@
+package amarja;
+import java.util.Scanner;
+/*Create a class Addition and write add method for:a) add two integers b) add two decimal c) add two String*/
+public class Addition {
+	int add (int no1,int no2)
+	{
+		return no1+no2;
+	}
+	double add (double no1,double no2)
+	{
+		return no1+no2;
+	}
+	String add (String s1,String s2)
+	{
```

Comment:
here we need to parse the string value into Integer and then convert it to string value and return that value.